aten/src/ATen/native/cpu/ReduceAllOpsKernel.cpp

```diff
@@ -99,7 +99,7 @@ static void max_all_kernel_impl(Tensor& result, const Tensor& input) {
     reduce_all_impl<int64_t>(result, input, lower_bound<int64_t>(),
       [=](int64_t a, int64_t b) -> int64_t { return max_impl(a, b); });
   } else {
-    AT_DISPATCH_ALL_TYPES(input.scalar_type(), "max_all", [&] {
+    AT_DISPATCH_ALL_TYPES_AND2(kHalf, kBFloat16, input.scalar_type(), "max_all", [&] {
```

Comment:
bool is supported by CUDA for its corresponding op, but not here for CPU. Is that intentional?

Reply:
Oops, bool has been handled above.